nautobot/extras/models/models.py

```diff
@@ -629,7 +630,7 @@ class ConfigContextSchema(OrganizationalModel):
 
     name = models.CharField(max_length=200, unique=True)
     description = models.CharField(max_length=200, blank=True)
-    slug = models.SlugField()
+    slug = AutoSlugField(populate_from="name", editable=True, overwrite_on_add=False)
```

Comment:
Config context schemas are a new enough feature that I'd be tempted to add `unique=True` to this one - IMHO that's an oversight.

Reply:
Sure, I just wanted to make sure the API didn't change.